web/src/invite.js

```diff
@@ -263,6 +282,10 @@ function open_invite_user_modal(e) {
         set_custom_time_inputs_visibility();
         set_expires_on_text();
 
+        $("#invite-user-modal").on("click", ".invite-tips-container a", () => {
+            dialog_widget.close_modal();
+        });
```

Comment:
Maybe add a comment that we're not doing `preventDefault` here as those links also do some additional action.